app/feature/feature-home/src/main/kotlin/com/hedvig/android/feature/home/home/data/GetHomeDataUseCase.kt

```diff
@@ -65,43 +61,11 @@ internal class GetHomeDataUseCaseImpl(
             link = it.link,
           )
         }
-        val commonClaimsData: List<CommonClaimsData> =
```

Comment:
Do we want to not have this information in the home screen anymore according to the new design?

Reply:
No, we removed the other services bottom sheet and now have button to the help center instead.